### ISSUE TITLE
Clips show unwanted subtitles 

### ISSUE THREAD

**Issue by PatHightree** [ewoudwijma/ActionCamVideoCompanion#3]
When loading back a project that has been rendered out, playing the clips show unwanted subtitles.
![image](https://user-images.githubusercontent.com/260904/69492035-139e0980-0e9d-11ea-8cd6-032f1634bd12.png)

**Comment by ewoudwijma** [comment]
Looks like encoder (klite?) loads the clips file which is stored in subtitle / srt format. Will investigate...

**Comment by ewoudwijma** [comment]
![image](https://user-images.githubusercontent.com/1737159/69537945-7537a400-0f81-11ea-84bc-da4d3232a684.png)

Do you have LAV splitter in your system tray? (installed by KLite I guess). What are your settings for subtitles there? See screenshot for what I have (I have no subtitles in acvc).

**Comment by ewoudwijma** [comment]
Check also KLite codec tweak tool
![image](https://user-images.githubusercontent.com/1737159/69538271-36eeb480-0f82-11ea-9572-715ecf684f8e.png)

Press button Preferred splitters:
![image](https://user-images.githubusercontent.com/1737159/69538216-132b6e80-0f82-11ea-907c-36005e5fcb37.png)

**Comment by PatHightree** [comment]
Nope, no LAV in my system tray but there was this green arrow VSFilter icon, I told it to hide subs and they went away in ACVC.
![image](https://user-images.githubusercontent.com/260904/69555576-aaed8480-0fa3-11ea-80d9-057a5b9431dd.png)

**Comment by ewoudwijma** [comment]
I propose to consider this a workaround for now. Future improvement is to have the codecs as part of the installation of acvc, most likely this will be libav but some more investigation needed
